Dockerfile

```diff
@@ -27,7 +29,18 @@ WORKDIR /gocode/src/github.com/replit/prybar
 
 RUN which node
 
-RUN make prybar-python2 prybar-python3 prybar-ruby prybar-lua prybar-spidermonkey prybar-nodejs
+# OCaml / Reason stuff
+RUN add-apt-repository ppa:avsm/ppa && \
+  apt-get update && \
+  apt-get install opam -y && \
+  opam init -y --disable-sandboxing && \
+  eval `opam env` && \
+  echo "eval \`opam env\`" >> ~/.bashrc && \
```

Comment:
Is there any way we can get away with not messing with ~/.bashrc.   The conditions under when it is and isn't not run are kinda nebulous.

Reply:
Right, I share your doubts... it really depends on how the docker container is being deployed and used though. I tried .profile as well, but for some reason bash didn't source .profile in my case.